commands/verify.go

```diff
@@ -100,16 +103,27 @@ func (v *VerifyCmd) AuthorizedKeysCommand(ctx context.Context, userArg string, t
 	if err != nil {
 		return "", err
 	}
+
 	if pkt, err := cert.VerifySshPktCert(ctx, v.PktVerifier); err != nil { // Verify the PKT contained in the cert
 		return "", err
-	} else if err := v.CheckPolicy(userArg, pkt, certB64Arg, typArg); err != nil { // Check if username is authorized
-		return "", err
-	} else { // Success!
-		// sshd expects the public key in the cert, not the cert itself. This
-		// public key is key of the CA that signs the cert, in our setting there
-		// is no CA.
-		pubkeyBytes := ssh.MarshalAuthorizedKey(cert.SshCert.SignatureKey)
-		return "cert-authority " + string(pubkeyBytes), nil
+	} else {
```

Comment:
remove else { ... }, since if returns

Reply:
The else is doing some lifting here. If I remove the else, the pkt will be out of scope since pkt is declared in the IF block